Templates/DefaultProject/Template/cmake/Platform/Linux/CompilerSettings.cmake

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+if(NOT CMAKE_C_COMPILER AND NOT CMAKE_CXX_COMPILER)
+    set(path_search
+        /bin
+        /usr/bin
+        /usr/local/bin
+        /sbin
+        /usr/sbin
+        /usr/local/sbin
+    )
+    list(TRANSFORM path_search APPEND "/clang-[0-9]*")
```

Comment:
Technically this regex could be `/clang-[1-9]+`

Reply:
What about `clang-10` though?  Maybe `clang-[0-9]+` instead.

Reply:
Well `/clang-[1-9]+` still matches "**clang-1** in "clang-10"

Reply:
should I make the "-" optional?
it also looks like clang 9 was `clang-9.0`, so I have to add the dot there

Reply:
actually, this is not feeding a regex but a glob search, I am just trying to differentiate it from `clang-cpp-12`